rails-snow-queen/db/migrate/20181201233429_create_quotes.rb

```diff
@@ -0,0 +1,15 @@
+class CreateQuotes < ActiveRecord::Migration[5.2]
+  def change
+    create_table :quotes do |t|
+      t.string :address
+      t.float :area
+      t.decimal :total
+      t.string :email
```

Comment:
Minor, but probably want an index on `email`

Reply:
Done ... I understand this is for faster searches, but why is this better than using the id?

Reply:
When you search for a quote, the thing you will likely know is the customer's ID, not their quote. You'll probably want to find all quotes for a given email